src/BUILD

```diff
@@ -280,6 +280,10 @@ cc_library(
         "status.hpp",
         "stringutils.cpp",
         "stringutils.hpp",
+        "systeminfo.cpp",
+        "systeminfo.hpp",
```

Comment:
Do we really need split the implementation into 2 cpp units?

Reply:
It is clear now which header you shall not include, and which functions not to call. I need to expose implementations for test to verify them. Otherwise I would have kept them in systeminfo.cpp.